.eleventy.js

```diff
@@ -1,6 +1,21 @@
 const sass = require('sass');
+const markdownIt = require('markdown-it');
```

Comment:
Do you need a dep for this too?

Reply:
It's included via 11ty. I could add it explicitly, I guess, but it's not technically needed.